drivers/w1/w1_ds248x.c

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2022 Caspar Friedrich <c.s.w.friedrich@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "w1_ds248x.h"
+
+static int ds248x_write(const struct i2c_dt_spec *spec, uint8_t cmd, uint8_t *data)
+{
+	int ret;
+
```

Comment:
No need for an empty line here

Reply:
I like to keep this helper variable separated from the actual logic. But no super strong opinion here.